CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Current Develop Branch
 
 ## 9.2.1 Thu Mar 25 2021
+- [#10692](https://github.com/MetaMask/metamask-extension/pull/10692): Prevent users from hitting the missing template error
```

Comment:
This might be a bit clearer:
```suggestion
- [#10692](https://github.com/MetaMask/metamask-extension/pull/10692): Prevent UI crash when a 'wallet_requestPermissions" confirmation is queued behind a "wallet_addEthereumChain" confirmation
```